autogen/oai/completion.py

```diff
@@ -185,28 +208,31 @@ def _get_response(cls, config: Dict, raise_on_ratelimit_or_timeout=False, use_ca
         """
         config = config.copy()
         openai.api_key_path = config.pop("api_key_path", openai.api_key_path)
+
+        model_name = config.get("model")
+        mapped_model_name = cls.llm_lite_mapping.get(model_name, model_name)  # Fallback to original if not found
+        config["model"] = mapped_model_name
+
         key = get_key(config)
         if use_cache:
```

Comment:
lines 190 to 199 weredeleted.
But line 189, the if statement, was not deleted?! 
Sure about that?

Reply:
Ah ...got it ... the old line 195 confused me. My mistake!
the if in line 189 "wrappes" the ne try block!
My fault! Plse. ignore the above comment.